model/src/DA/Finance/Asset/Settlement.daml

```diff
@@ -63,6 +63,7 @@ template AssetSettlementRule
         do
```

Comment:
I think since we only merge this to the `marketplace` branch for now, it's safe to remove this file alltogether.